src/bun.js/bindings/JSBundlerPlugin.h

```diff
@@ -130,12 +141,21 @@ class BundlerPlugin final {
         this->config = config;
     }
 
+    ~BundlerPlugin()
+    {
+        if (virtualModules) {
+            delete virtualModules;
+        }
+    }
+
     NamespaceList onLoad = {};
     NamespaceList onResolve = {};
     NativePluginList onBeforeParse = {};
     BunPluginTarget target { BunPluginTargetBrowser };
 
     WriteBarrierList<JSC::JSPromise> deferredPromises = {};
+    VirtualModuleMap* virtualModules { nullptr };
```

Comment:
Don't make this a pointer.
```suggestion
    VirtualModuleMap virtualModules { nullptr };
```